metal/roles/pxe_server/files/docker-compose.yml

```
@@ -2,18 +2,16 @@ version: "3"
 
 services:
   dhcp:
```

Missing name change:

```suggestion
  dnsmasq:
```